public/style/bzlite.css

```diff
@@ -935,3 +935,23 @@ input[type=search]:focus + .focus {
 #navigation a:hover {
   background: #333;
 }
+
+div.input {
+  position: relative;
+}
+
+div.cleanInput {
+  position: absolute;
+  float: right;
+  top: -0.5rem;
```

Comment:
This is misplacing the remove icon for me, `0rem` is working on desktop fine though